### DISCUSSION TITLE
Can I use opensearch instead elasticsearch like Index engine with janusgraph and thehive??

### DISCUSSION THREAD

**Discussion by romarito90** [JanusGraph/janusgraph#3908] [Q&A]
Hello everyone

My question is: can I use opensearch instead elasticsearch with janusgraph and thehive??

Thehive needs an storage backend like Cassandra and Index engine like lucene or elasticsearch. So I want to know if someone use this combination like thehive + Cassandra + opensearch. I SEE the documentation of janusgraph and opensearch doesnt appear in the Index backend options.

**Comment by hadoopmarc** [comment]
There is no support for opensearch, but others seemed to have some luck with it, see: https://github.com/JanusGraph/janusgraph/discussions/2837

**Reply by romarito90** [reply]
Thank you so much for your reply @hadoopmarc 

regards
